MAUI/TimePicker/overview.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: About .NET MAUI Time Picker control | Syncfusion
+description: Learn here about the overview of Syncfusion .NET MAUI Time Picker (SfTimePicker) control, its basic features, and time picker functionalities.
+platform: maui
+control: SfTimePicker
+documentation: ug
+---
+
+# Overview of .NET MAUI Time Picker (SfTimePicker)
+
+Syncfusion .NET MAUI Time Picker (SfTimePicker) control allows you to select time and visualizes its items inside a popup or a drop-down UI element. The Time Picker also enables you to set time formats, and to customize its dialog appearance by configuring its header, footer, and more.
+
+## Key features
+
+* **Header view**: The Time Picker allows you to add and customize header text.
```

Comment:
* **Header view**: The Time Picker allows you to add and customize the header text.

Reply:
Content modified.